pkg/collector/container.go

```diff
@@ -57,23 +57,38 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 	if argsMap == nil {
 		argsMap = map[string]string{}
 	}
+	// defines the output (sorted) array for final output
+	var args []string
+	// When adding a config via v1alpha1.OpenTelemetryCollectorSpec.Config, we ensure that it is always the
+	// first item in the args. At the time of writing, although multiple configs are allowed in the
+	// opentelemetry collector, the operator has yet to implement such functionality.  When multiple configs
+	// are present they should be merged in a deterministic manner using the order given, and because
+	// v1alpha1.OpenTelemetryCollectorSpec.Config is a required field we assume that it will always be the
+	// "primary" config and in the future additional configs can be appended to the container args in a simple manner.
+	// Note that ordering is only important for config and not other arguments such as feature gates or etc.
```

Comment:
```suggestion
	// Note that ordering is presently only important for config and not other arguments such as feature gates or etc.
```

Or this line can be removed entirely.  More arguments may be added in the future and they may be order dependent.

Reply:
Thanks for the comment.  I have deleted the line.  You are right that we may have other order dependent args in the future.